src/kv/store.h

```diff
@@ -101,6 +101,9 @@ namespace kv
     // If true, use historical ledger secrets to deserialise entries
     const bool is_historical = false;
 
+    // Ledger entry header flags
+    uint8_t flags = 0;
```

Comment:
This needs to be cleared on rollback.

Reply:
See line 615; I was conservative and roll back only the chunking flag instead of all of them.

Reply:
Thanks, I'd missed that!